portfolio.css

```diff
@@ -81,18 +82,29 @@ nav .container {
 	overflow: auto;
 }
 
+.brushstroke {
+	background: url("brushstroke.png");
+	background-repeat: no-repeat;
+	background-size: 100% 100%;
```

Comment:
No need to have 100% twice, you can just put it once if the x and y sizes are the same.